src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 </Intro>
 
 <YouWillLearn>
 
-* What the most popular editors are
-* How to format your code automatically
+* સૌથી વધુ લોકપ્રિય એડિટરો શું છે
+* તમારો કોડ આપમેળે કેવી રીતે ફોર્મેટ કરવો
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## તમારુ એડિટર {/*your-editor*/}
 
-[VS Code](https://code.visualstudio.com/) is one of the most popular editors in use today. It has a large marketplace of extensions and integrates well with popular services like GitHub. Most of the features listed below can be added to VS Code as extensions as well, making it highly configurable!
+[VS Code](https://code.visualstudio.com/) એ આજે ઉપયોગમાં લેવાતા સૌથી લોકપ્રિય એડિટરો પૈકીનું એક છે. તેની પાસે એક્સ્ટેંશનનું વિશાળ બજાર છે અને GitHub જેવી લોકપ્રિય સેવાઓ સાથે સારી રીતે સંકલિત છે. નીચે સૂચિબદ્ધ મોટાભાગની સુવિધાઓ VS Code માં એક્સ્ટેંશન તરીકે પણ ઉમેરી શકાય છે, જે તેને અત્યંત રૂપરેખાંકિત બનાવે છે!
 
-Other popular text editors used in the React community include:
+React સમુદાયમાં વપરાતા અન્ય લોકપ્રિય ટેક્સ્ટ એડિટરો માં નીચેના એડિટરો નો સમાવેશ થાય છે:
 
-* [WebStorm](https://www.jetbrains.com/webstorm/) is an integrated development environment designed specifically for JavaScript.
-* [Sublime Text](https://www.sublimetext.com/) has support for JSX and TypeScript, [syntax highlighting](https://stackoverflow.com/a/70960574/458193) and autocomplete built in.
-* [Vim](https://www.vim.org/) is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as "vi" with most UNIX systems and with Apple OS X.
+* [WebStorm](https://www.jetbrains.com/webstorm/) એક સંકલિત વિકાસ પર્યાવરણ છે જે ખાસ કરીને JavaScript માટે રચાયેલ છે.
+* [Sublime Text](https://www.sublimetext.com/) JSX અને TypeScript સપોર્ટ કરે છે,[સિન્ટેક્સ હાઇલાઇટિંગ](https://stackoverflow.com/a/70960574/458193) અને સ્વતઃપૂર્ણ બિલ્ટ ઇન માટે સપોર્ટ ધરાવે છે.
+* [Vim](https://www.vim.org/) એ અત્યંત રૂપરેખાંકિત ટેક્સ્ટ એડિટર છે જે કોઈપણ પ્રકારના ટેક્સ્ટને ખૂબ જ કાર્યક્ષમ બનાવવા અને બદલવા માટે બનાવવામાં આવ્યું છે. તે મોટાભાગની UNIX સિસ્ટમો અને Apple OS X સાથે vi તરીકે સમાવિષ્ટ છે.
 
-## Recommended text editor features {/*recommended-text-editor-features*/}
+## ટેક્સ્ટ એડિટર ની સુવિધાવો માટેની ભલામણ {/*recommended-text-editor-features*/}
 
-Some editors come with these features built in, but others might require adding an extension. Check to see what support your editor of choice provides to be sure!
+કેટલાક એડિટરો બિલ્ટ-ઇન આ સુવિધાઓ સાથે આવે છે, પરંતુ અન્યને એક્સ્ટેંશન ઉમેરવાની જરૂર પડી શકે છે. ખાતરી કરવા માટે તમારા પસંદગીના એડિટર શું સમર્થન આપે છે તે જોવા માટે તપાસો!
 
-### Linting {/*linting*/}
+### લિંટિંગ {/*linting*/}
 
-Code linters find problems in your code as you write, helping you fix them early. [ESLint](https://eslint.org/) is a popular, open source linter for JavaScript. 
+કોડ લિંટર્સ તમારા કોડમાં સમસ્યાઓ શોધી કાઢે છે જેમ તમે લખો છો, જે તમને વહેલી તકે ઠીક કરવામાં મદદ કરે છે. [ESLint](https://eslint.org/) એ JavaScript માટે લોકપ્રિય, ઓપન સોર્સ લિંટર છે. 
```

Comment:
Change every Javascript to જાવાસ્ક્રિપ્ટ. Because it already mentioned in glossary list.